clients/admin-ui/src/features/system/SystemFormTabs.tsx

```diff
@@ -69,12 +69,14 @@ const ToastMessage = ({
 };
 
 const SystemFormTabs = ({
+  initialTabIndex = 0,
```

Comment:
Adding this to allow the parent component to set the initial tab, subsequent tab changes are still handled by this component.